rust/data/presets.json

```diff
@@ -52,46 +54,46 @@
       "canGateGlitch",
       "canOffScreenSuperShot",
       "canGravityJump",
+      "canLateralMidAirMorph",
       "canMockball",
       "canSpringBallJumpMidAir",
-      "canUseFrozenEnemies",
       "canMochtroidIceClimb",
       "canMoonwalk",
       "canMoonfall",
       "canMochtroidIceClip",
       "canCeilingClip",
       "canIframeSpikeJump",
       "canBombHorizontally",
-      "canSnailClimb",
       "canCrumbleJump",
       "canCrystalFlash",
       "canConsecutiveWalljump",
       "canCrossRoomJumpIntoWater",
       "canShinechargeMovement",
       "canMidairShinespark",
-      "canCarefulJump",
-      "canXrayWaitForIFrames",
+      "canXRayWaitForIFrames",
       "canDownBack",
       "canPlayInSand",
       "canHyperGateShot",
       "canSpaceJumpWaterBounce",
       "canWaterShineCharge",
-      "canHeroShot"
+      "canHeroShot",
+      "canMaridiaTubeClip"
```

Comment:
I'm thinking tube clip is obscure enough to be in VH

Reply:
Obscure enough to not put in Hard, Trivial enough to not add to VeryHard 🤣 
I can leave it in VH